fastlane/lib/fastlane/command_line_handler.rb

```diff
@@ -30,8 +30,12 @@ def self.handle(args, options)
 
       if FastlaneCore::FastlaneFolder.swift?
         disable_runner_upgrades = options.disable_runner_upgrades || false
-
-        Fastlane::SwiftLaneManager.cruise_lane(lane, lane_parameters, dot_env, disable_runner_upgrades: disable_runner_upgrades)
+        swift_server_port = options.swift_server_port
+        if swift_server_port
+          Fastlane::SwiftLaneManager.cruise_lane(lane, lane_parameters, dot_env, disable_runner_upgrades: disable_runner_upgrades, swift_server_port: swift_server_port)
+        else
+          Fastlane::SwiftLaneManager.cruise_lane(lane, lane_parameters, dot_env, disable_runner_upgrades: disable_runner_upgrades)
+        end
```

Comment:
We can get rid of the `if swift_server_port` and just make this... (see notes from other comment)

```rb
swift_server_port = options.swift_server_port
Fastlane::SwiftLaneManager.cruise_lane(lane, lane_parameters, dot_env, disable_runner_upgrades: disable_runner_upgrades, swift_server_port: swift_server_port)
```